spec/user_repository_spec.rb

```diff
@@ -0,0 +1,45 @@
+require 'user'
+require 'user_repository'
+
+def reset_users_table
+  seed_sql = File.read('spec/seeds/users_seeds.sql')
+  connection = PG.connect({ host: '127.0.0.1', dbname: 'chitter-challenge' })
+  connection.exec(seed_sql)
+end 
+
+describe UserRepository do
+  before(:each) do 
+      reset_users_table
+  end
+  it 'returns all users'do
+    repo = UserRepository.new
+    users = repo.all
+    expect(users.length).to eq(2)
+    expect(users.first.name).to eq("sara")
+  end
+
+
+  it ' find specific user' do
+    repo = UserRepository.new
+
+    user = repo.find(2)
+    expect(user.id).to eq(2)
+  end
```

Comment:
Have a look at the 'Creating and signing in users' John posted in the July channel on Friday. It tells you a lot about how to approach this, because the next step will be to login a user and you will need to find the user based on their email, not their id.